File: pascal/parité2.pas
program PariteEntreDixNombres;

var N : array[1..10] of integer;
var I, Np, Ni : integer;

begin
  writeln('Veulliez entrer dix nombre : ');
  for I := 1 to 10 do
  begin
    write('Nombre ', I, ': ');
    readln(N[I]);
  end;
  
  Ni := 0;
  Np := 0;
  for I := 1 to 10 do
  begin
    if N[I] mod 2 = 0 then
    begin
      Np := Np + 1;
    end
    else
    begin
      Ni := Ni + 1;
    end  
  end;   
  write('Les nombres pairs sont ', Np);
  writeln(' et les nombres impairs sont ', Ni);
end.    
        

File: python/parité2.py
N = int(input("Entrer un nombre entier positif : "))

print(f"Les nombres pairs compris entre 1 à {N} sont : ", end=" ")
for I in range(1, N + 1):
    if I % 2 == 0:
        print(I, end=" ")

input()        
